Resources/Views/categories.leaf

```diff
@@ -3,97 +3,25 @@
-<span class="input-group-btn" style="margin-right: 35px;">
-<button type="submit" class="btn btn-default" type="button">
-<span class="glyphicon glyphicon-search"></span>
-</button>
-</span>
-</form>
-</div>
-<br>
-<p class="lead">Categories</p>
-<div class="list-group">
-<a href="/accessories" class="list-group-item">All Accessories
-<span class="badge">#(accessoryCount)</span>
-</a>
-<a href="/manufacturers" class="list-group-item">All Manufacturers
-<span class="badge">#(manufacturerCount)</span>
-</a>
-</div>
-<div class="btn-group btn-block visible-sm visible-xs">
-<button type="button" class="btn btn-default btn-block dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">Choose Category
-<span class="caret"></span>
-</button>
-<ul class="dropdown-menu btn-block">
-#for(category in categories) {
-<li>
-<a href="/categories/#(category.id)" class="dropdown-item btn-block #if(index == selectedCategory) { selected }">
-<img src="#(category.image)" width="30px"> #(category.name)
-<span class="badge badge-important pull-right">#(category.accessories_count)</span>
-</a>
-</li>
-<li role="separator" class="divider"></li>
-}
-</ul>
-</div>
-<div class="list-group hidden-sm hidden-xs">
-#for(category in categories) {
-<a href="/categories/#(category.id)" class="list-group-item #if(index == selectedCategory) { selected }">
-<img src="#(category.image)" width="30px" height="30px"> #(category.name)
-<span class="badge">#(category.accessories_count)</span>
-</a>
-}
-</div>
-<div id="email-subscription">
-<p class="lead" style="margin-bottom: 10px;">Keep up to date</p>
-<p>Receive new accessories in your inbox once per month.</p>
-<div class="input-group" style="display: block;">
-<form action="https://world.us16.list-manage.com/subscribe/post?u=87ff0444f06c586f363e15184&amp;id=e5c558d68d" method="post"
-id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" style="display: flex;"
-novalidate>
-<input type="email" value="" class="require email form-control" id="mce-EMAIL" name="EMAIL" placeholder="info@homekitty.world">
-<span class="input-group-btn" style="margin-right: 35px;">
-<button type="submit" value="Subscribe" name="subscribe" id="" class="button btn btn-default" type="button">
-<span class="glyphicon glyphicon-send"></span>
-</button>
-</span>
-</form>
-<div id="mce-responses" class="clear" style="padding: 5px;">
-<div class="response" id="mce-error-response" style="display: none"></div>
-<div class="response" id="mce-success-response" style="display: none"></div>
-</div>
-</div>
-</div>
-</div>
+    <div class="row">
 
-<!-- Content -->
-<div class="col-md-9">
-<div class="flex-item" style="margin-top: 10px;">
-<img src="#get(pageIcon)" width="50px">
-<p class="lead">#(category.name)</p>
-</div>
-<div class="row">
-#for(accessory in accessories) {
-#embed("accessoryItem")
-}
-</div>
-</div>
-</div>
-</div>
-}
+        <!-- Side Menu -->
+        #embed("sidemenu")
```

Comment:
I removed this in the past because of some issues in the loading of the HTML if I remember correctly. When the page was still loading there was a little annoying jump in Safari, we can try to see if it's fixed now in staging by merging this PR.